### PR TITLE
Should fix some windows service issues

### PR DESCRIPTION
**Pull request by tewinget**

### CODE REVIEW

**Review by despair86 — APPROVED** [review]

File: daemon/main.cpp [hunk 1]
```diff
@@ -290,6 +290,25 @@ run_main_context(const fs::path confFile, const llarp::RuntimeOptions opts)
   }
 }
 
+#ifdef _WIN32
+void
+TellWindowsServiceStopped()
+{
+  ::WSACleanup();
+  ReportSvcStatus(SERVICE_STOPPED, NO_ERROR, 0);
```
Comment (despair86) [inline daemon/main.cpp]: this could return the exit code from `lokinet_main()`

**Review by majestrate — APPROVED** [review]

**Comment by tewinget** [comment]
PR is still incomplete, was initially made to trigger CI.

**Comment by despair86** [comment]
i might be terrible at times but the tun code is one of the less cursed things about the platform-specific hooks pls restore

**Comment by tewinget** [comment]
> i might be terrible at times but the tun code is one of the less cursed things about the platform-specific hooks pls restore

yeah, this PR was for triggering CI and went out of scope.  closing.

**Pull request closed without merging**